modules/vba_documents/spec/sidekiq/upload_upstream_processing_error_batch_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe VBADocuments::UploadUpstreamProcessingErrorBatch, type: :job do
+  let(:client_stub) { instance_double(CentralMail::Service) }
+
+  # test target, less than 14 days old in emms error status, job should pick up this record
+  let!(:emms_error_upload) { create(:upload_submission, :status_emms_error) }
+
+  # Old errored record, job should not pick up since it's older than 14 days
+  let!(:old_error_upload) { create(:upload_submission, :status_emms_error, created_at: 20.days.ago) }
```

Comment:
Similar comment to above, recommend using the `MAX_UPSTREAM_ERROR_AGE_DAYS` constant here to avoid enshrining 14 days in the spec.

Reply:
fixed